### ISSUE TITLE
[DOCUMENTATION] Custom chain from .env

### ISSUE THREAD

**Issue by ZAlexanderV** [cosmos/cosmos-multisig-ui#251]
Is it possible to add custom chains from .env file and disable cosmos registry?
Can anyone provide an example .env file for a custom network?

**Comment by abefernan** [comment]
Yes. You can set `NEXT_PUBLIC_MULTICHAIN=false` and that will disable the chain selection menu. Data will be queried from the chain-registry, still. In order to use a custom network you can also set some more variables on the envfile, you can see them in [this function](https://github.com/cosmos/cosmos-multisig-ui/blob/main/context/ChainsContext/storage.tsx#L144-L186) prefixed with `NEXT_PUBLIC_` (this is not documented anywhere, sorry about that!).

**Comment by ZAlexanderV** [comment]
I constructed .env.local with values from function, but seems code is broken.
Prefix, Chain name are not applied and not working.

**Comment by abefernan** [comment]
Have you tried navigating directly to `<url>/<my-registry>`, where `url` would be your localhost or wherever you are hosting it, and `my-registry` would be the value of `NEXT_PUBLIC_REGISTRY_NAME` in your envfile (and not named like another chain from the registry)? I tried that and it loaded the values correctly
